code/game/objects/items/storage/dakis.dm

```diff
@@ -0,0 +1,44 @@
+//////////////////////////////////
+ //dakimakuras
+ //////////////////////////////////
+
+/obj/item/storage/daki
+	name = "dakimakura"
+	desc = "A large pillow depicting a girl in a compromising position. Featuring as many dimensions as you."
+	icon = 'icons/obj/daki.dmi'
+	icon_state = "daki_base"
```

Comment:
Need to define the inhand location.

lefthand_file = 'icons/mob/inhands/misc/daki_lefthand.dmi'
righthand_file = 'icons/mob/inhands/misc/daki_righthand.dmi'

Reply:
Sure I'll do it in the morning though

Reply:
Added you can close this I guess